client/app/js/controllers/receiver.js

```diff
@@ -7,7 +7,12 @@ GLClient.controller('ReceiverSidebarCtrl', ['$scope', '$location', function($sco
 
 GLClient.controller('ReceiverTipsCtrl', ['$scope',  '$http', '$route', '$location', '$uibModal', 'ReceiverTips',
   function($scope, $http, $route, $location, $uibModal, ReceiverTips) {
-  $scope.tips = ReceiverTips.query();
+  $scope.tips = ReceiverTips.query(function(tips) {
+      angular.forEach(tips ,function(tip) {
+          tip.export = '/rtip/' + tip.id + '/export';
+        });
+      return tips;
+  });
```

Comment:
Make the change. The attribute on the form needs to be: `action="'/rtip/{{::tip.id }}/export"`

I carelessly thought that string concatenation of this form was not the style.